### PR TITLE
[wdspec] Align aborted navigation tests with specification

### PR DESCRIPTION
**Pull request by sadym-chromium**
Align aborted navigation tests with specification. 

WebDriver BiDi requires to succeed the navigation, if it was aborted by another navigation:
* [`browsingContext.navigate`](https://w3c.github.io/webdriver-bidi/#command-browsingContext-navigate)
* [Aawait a navigation](https://w3c.github.io/webdriver-bidi/#await-a-navigation):
* [Navigate](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigate) navigable...
* [navigate](https://html.spec.whatwg.org/#navigate), step 22.3: [abort a document and its descendants](https://html.spec.whatwg.org/#abort-a-document-and-its-descendants) given navigable's [active document](https://html.spec.whatwg.org/#nav-document).
* [Abort a document and its descendants](https://html.spec.whatwg.org/#abort-a-document-and-its-descendants):
* [Abort a document](https://html.spec.whatwg.org/#abort-a-document)
* If document's [during-loading navigation ID for WebDriver BiDi](https://html.spec.whatwg.org/#concept-document-navigation-id) is non-null, then:
* Invoke [WebDriver BiDi navigation aborted](https://w3c.github.io/webdriver-bidi/#webdriver-bidi-navigation-aborted) (emits `browsingContext.navigationAborted` event).
* This case does not fall under **_"If event received is "navigation failed" return [error](https://w3c.github.io/webdriver/#dfn-errors) with [error code](https://w3c.github.io/webdriver/#dfn-error-code) [unknown error](https://w3c.github.io/webdriver/#dfn-unknown-error)"_**
* The command should **_"Return [success](https://w3c.github.io/webdriver/#dfn-success) with data body."_**

### CODE REVIEW

File: webdriver/tests/bidi/browsing_context/navigate/navigate_tentative.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+async def test_interactive_change_location_succeeded(bidi_session, inline,
+      new_tab):
+    new_url = inline("<div>foo</div>")
+    url_with_redirect = inline(f"<script>window.location='{new_url}';</script>")
+    result = await bidi_session.browsing_context.navigate(
+        context=new_tab["context"], url=url_with_redirect, wait="interactive")
+
+    any_string(result["navigation"])
+    assert result["url"] == new_url
```
Comment (juliandescottes) [inline webdriver/tests/bidi/browsing_context/navigate/navigate_tentative.py:29]: By definition, one of the two tests will necessarily fail. Given that both chrome and firefox pass test_interactive_change_location_throws at the moment, maybe we should only keep this one and updated when the spec is clarified?

**Review by juliandescottes — CHANGES_REQUESTED** [review]
I was looking at the two similar tests in bidi/browsing_context/navigation_failed/navigation_failed.py, at the moment they expect a `navigationFailed` event for similar scenarios. If I understand the spec change correctly, this also wrong? Can you check if they need updating?

**Comment by sadym-chromium** [comment]
> I was looking at the two similar tests in bidi/browsing_context/navigation_failed/navigation_failed.py, at the moment they expect a `navigationFailed` event for similar scenarios. If I understand the spec change correctly, this also wrong? Can you check if they need updating?

I'm working on it from the implementation side: https://github.com/GoogleChromeLabs/chromium-bidi/issues/2856

**Comment by juliandescottes** [comment]
> > I was looking at the two similar tests in bidi/browsing_context/navigation_failed/navigation_failed.py, at the moment they expect a `navigationFailed` event for similar scenarios. If I understand the spec change correctly, this also wrong? Can you check if they need updating?
> 
> I'm working on it from the implementation side: [GoogleChromeLabs/chromium-bidi#2856](https://github.com/GoogleChromeLabs/chromium-bidi/issues/2856)

Does this mean you will also update those tests here or will it be another PR?

**Comment by sadym-chromium** [comment]
closing in favor of https://github.com/web-platform-tests/wpt/pull/49718

**Pull request closed without merging**